test/controllers/api/release_controller_test.exs

```diff
@@ -87,7 +87,7 @@ defmodule HexWeb.API.ReleaseControllerTest do
     assert conn.status == 422
     body = Poison.decode!(conn.resp_body)
     assert body["message"] == "Validation error(s)"
-    assert body["errors"]["meta"]["links"] == "expected type dict(string, string)"
+    assert body["errors"]["meta"]["links"] == "is invalid"
```

Comment:
it looks like by using embeds it's relying on ecto "type-checking" and so the errors are always "is invalid" when there's a type mismatch. Not sure if there's a way to overwrite this behaviour. Perhaps it's good enough to have it like this?